README.md

```diff
@@ -74,6 +74,14 @@ I18n.backend = Pseudolocalization::I18n::Backend.new(I18n.backend)
 I18n.backend.ignores = ['ignored*', /Waldo.$/]
 ```
 
+### Apply only to specific locales
+You may wish to have the backend only apply to specific locales. These may be configured via an array, `only_locales`, on the backend.
+
+```ruby
+I18n.backend = Pseudolocalization::I18n::Backend.new(I18n.backend)
+I18n.backend.only_locales = [:en, :xx, :zz]
```

Comment:
I wonder if it wouldn't be clearer if we were to reuse the naming convention of `available_locales`. If the pseudolocalization backend's `available_locales` doesn't include locale X, then it passes it through?